jobs/payment-jobs/requirements.txt

```diff
@@ -1,26 +1,46 @@
+-e git+https://github.com/bcgov/sbc-common-components.git@dd41940a5465750bfed5c5fe1b06d3375ca13dcd#egg=sbc_common_components&subdirectory=python
```

Comment:
I read the order doesn't matter in this file, it builds a dependency graph I believe